DEVELOPMENT.md

```diff
@@ -249,6 +213,24 @@ check the
 page to ensure that all services are up and running (and not blocked by a quota
 issue, for example).
 
+### Deploy Knative Ingress
+
+Knative supports Istio, Contour, and Kourier as the Ingress solution.
+
+For simplicity, you can just run the following command to install Kourier.
+```
+kubectl apply -f https://github.com/knative/net-kourier/releases/download/v0.18.0/kourier.yaml
+
+kubectl patch configmap/config-network \
+  -n knative-serving \
+  --type merge \
+  -p '{"data":{"ingress.class":"kourier.ingress.networking.knative.dev"}}'
+```
+
+Alternatively, you can follow the development instructions of either [Istio](https://github.com/knative-sandbox/net-istio/blob/master/DEVELOPMENT.md)
+or [Contour](https://github.com/knative-sandbox/net-contour/blob/master/DEVELOPMENT.md) to develop Knative
+against them respectively.
```

Comment:
Perhaps as a more general statement here, just redirect to the networking portion of our install docs?

Reply:
Done